applicationName.sh

```diff
@@ -1,59 +1,46 @@
 #!/bin/bash
+# shellcheck disable=SC2155,SC2001,SC2002,SC2154,SC1089
 set -e
 
 export HELM_ARGS=$1
 
 export FILES=""
 
-eval "array=($HELM_ARGS)";
-for i in "${!array[@]}"; do
-  value="${array[$i]}"
-  if [ "$value" == '-f' ]
-  then
-    export FILES="$FILES ${array[$i+1]}"
-  elif [[ "$value" == applicationName=* ]]
-  then
-    export APPLICATION_NAME=$(echo "$value" | sed 's/applicationName=//g')
-    echo $APPLICATION_NAME
+eval "ARRAY=($HELM_ARGS)"
+if [ "$value" == '-f' ]; then
+    export FILES="$FILES ${ARRAY[$i + 1]}"
+  elif [[ "$value" == applicationName=* ]]; then
+    export readonly APPLICATION_NAME=$(echo "$value" | sed 's/applicationName=//g')
+    echo "$APPLICATION_NAME"
     exit 0
-  elif [[ "$value" == application.name=* ]]
-  then
-    export APPLICATION_NAME=$(echo "$value" | sed 's/application.name=//g')
-    echo $APPLICATION_NAME
+  elif [[ "$value" == application.name=* ]]; then
+    export readonly APPLICATION_NAME=$(echo "$value" | sed 's/application.name=//g')
```

Comment:
Look here how to set readonly variables https://unix.stackexchange.com/questions/66627/is-there-anyway-to-set-a-readonly-environment-variable#:~:text=readonly%20VAR%3Dfoo%20%20%20%23%20POSIX

`readonly VAR=foo
`